Xamarin.Forms.Maps.Android/MapRenderer.cs

```diff
@@ -372,5 +357,23 @@ void UpdateVisibleRegion(LatLng pos)
 			double dlong = Math.Max(Math.Abs(ul.Longitude - lr.Longitude), Math.Abs(ur.Longitude - ll.Longitude));
 			Element.VisibleRegion = new MapSpan(new Position(pos.Latitude, pos.Longitude), dlat, dlong);
 		}
+
+		void IOnMapReadyCallback.OnMapReady(GoogleMap map)
+		{
+			NativeMap = map;
+			if (map == null)
+			{
+				return;
+			}
+
+			map.SetOnCameraChangeListener(this);
+			map.InfoWindowClick += MapOnMarkerClick;
+
+			map.UiSettings.ZoomControlsEnabled = Map.HasZoomEnabled;
+			map.UiSettings.ZoomGesturesEnabled = Map.HasZoomEnabled;
+			map.UiSettings.ScrollGesturesEnabled = Map.HasScrollEnabled;
+			map.MyLocationEnabled = map.UiSettings.MyLocationButtonEnabled = Map.IsShowingUser;
+			SetMapType();
+		}
```

Comment:
Could OnMapReady be made a protected virtual method, so that subclasses can do setup on the NativeMap as well?